code/datums/mind.dm

```diff
@@ -1215,6 +1226,42 @@
 				log_admin("[key_name(usr)] has automatically forged objectives for [key_name(current)]")
 				message_admins("[key_name_admin(usr)] has automatically forged objectives for [key_name_admin(current)]")
 
+	else if(href_list["brother"])
+		switch(href_list["brother"])
+			if("clear")
+				if(has_antag_datum(/datum/antagonist/brother))
+					to_chat("<span class='warning'><FONT size = 3><B>You have been brainwashed! You are no longer a blood brother!</B></FONT></span>")
+					remove_antag_datum(/datum/antagonist/brother)
+					log_admin("[key_name(usr)] has de-brothered [key_name(current)]")
+					message_admins("[key_name_admin(usr)] has de-brothered[key_name_admin(current)]")
+
+			if("brother")
+				if(!(has_antag_datum(/datum/antagonist/brother)))
+					var/list/candidates = list()
+					for(var/mob/living/L in GLOB.alive_mob_list)
+						if(!L.mind || L.mind == current)
```

Comment:
What if the candidate is already a blood brother? I'd probably filter them out from the get-go or warn the admin.